.github/workflows/ci-build.yaml

```diff
@@ -96,3 +96,5 @@ jobs:
     - name: Run tests
       run: |
         python -m pytest
+        # Also run train_test.py on 2 devices to check pmap works ok.
```

Comment:
In general, it looks like they try to name steps a little more thoroughly, so I would do:
- name: Run tests on single device
run: | 
python -m pytest
- name: Run train_test on 2 devices to check that pmap works
run: |
FERMINET_CHEX_N_CPU_DEVICES=2 python -m pytest ferminet/tests/train_test.py

Reply:
Good idea, thanks!

PTAL.